### DISCUSSION TITLE
<deleted>

### DISCUSSION THREAD

**Discussion by cookieJarYum** [pixijs/pixijs#7561] [Q&A]
<deleted>

**Comment by ivanpopelyshev** [comment]
You can re-use graphics geometry, `new Graphics(graphics1.geometry)` that'll be a little faster. however you need to switch off batching for this.

Instanced renderer for graphics with same geometry is definitely a thing, i know that several people coded it for their projects but no one shared. 

> Is there a better way to do this?

Draw each cluster with mesh-shader.
